cfep-21.md

```diff
@@ -0,0 +1,76 @@
+
+<table>
+<tr><td> Title </td><td> conda-forge / Bioconda Transfer Policy </td>
+<tr><td> Status </td><td> Draft </td></tr>
+<tr><td> Author(s) </td><td> Mervin Fansler &lt;mervin.fansler@bric.ku.dk&gt;</td></tr>
+<tr><td> Created </td><td> 10 July 2024</td></tr>
+<tr><td> Updated </td><td> 19 September 2024</td></tr>
+<tr><td> Discussion </td><td> https://github.com/conda-forge/cfep/pull/52 </td></tr>
+<tr><td> Implementation </td><td> NA </td></tr>
+</table>
+
+## Abstract
+
+Bioconda (BC) and conda-forge (CF) have occasionally transferred package recipes between them. The `strict` channel
+priority recommended to BC users and used when building packages on the BC channel poses an
+issue when transfers occur, especially in the CF-to-BC direction. This proposal aims to 
+define a policy for standard procedures when such transfers occur. Included in this is the introduction
+of a new label to be applied to all package builds that transfer CF-to-BC.
+
+## Specification
+
+### Bioconda to conda-forge (BC-to-CF) Transfers
+
+1. Packages on BC considered for transfer should be marked with a ["Move to conda-forge" label](https://github.com/bioconda/bioconda-recipes/labels/Move%20to%20Conda-Forge) in the `bioconda-recipes` repository. This label can be applied either in an Issue or a Pull Request.
+2. A Pull Request should be create on CF's `staged-recipes` repository to add the recipe. A link to the labeled BC Issue/Pull Request should be included in the body of the CF Pull Request.
+3. The linter will flag when adding a recipe already hosted by BC. This will trigger a message to [the **conda-forge/bioconda-recipes** team](https://github.com/orgs/conda-forge/teams/bioconda-recipes).
```

Comment:
This is an implementation detail we don't need in the specification. Perhaps in the Example workflow section.